lib/cocoapods/installer/analyzer.rb

```diff
@@ -921,6 +921,25 @@ def generate_sandbox_state(specifications)
         sandbox_state
       end
 
+      # @param  [Platform] platform
+      #         The platform to build against
+      #
+      # @return [Boolean] Whether the platform requires 64-bit architectures
+      #
+      def requires_64_bit_archs?(platform)
+        return false unless platform
+        case platform.name
+        when :osx
+          true
+        when :ios
+          platform.deployment_target >= Version.new('11.0')
```

Comment:
extract the version here into a constant? `IOS_64_BIT_ONLY_VERSION` or something?